Forge.Forms/src/Forge.Forms/Controls/DialogWindow.xaml.cs

```diff
@@ -1,32 +1,37 @@
-﻿using System.Windows.Input;
+﻿using System.Windows;
+using System.Windows.Input;
 using MahApps.Metro.Controls;
 
 namespace Forge.Forms.Controls
 {
-    /// <summary>
-    /// Interaction logic for MainWindow.xaml
-    /// </summary>
-    public partial class DialogWindow : MetroWindow
-    {
-        public DialogWindow(object model, object context, WindowOptions options)
-        {
-            DataContext = options;
-            InitializeComponent();
-            Loaded += (sender, e) => MoveFocus(new TraversalRequest(FocusNavigationDirection.Next));
-            Form.Environment.Add(options.EnvironmentFlags);
-            Form.Context = context;
-            Form.Model = model;
-        }
+	/// <summary>
+	/// Interaction logic for MainWindow.xaml
+	/// </summary>
+	public partial class DialogWindow : MetroWindow
+	{
+		public DialogWindow(object model, object context, WindowOptions options)
+		{
+			DataContext = options;
+			InitializeComponent();
+			Loaded += (sender, e) =>
+			{
+				MoveFocus(new TraversalRequest(FocusNavigationDirection.Next));
+			};
+			Form.Environment.Add(options.EnvironmentFlags);
+			Form.Context = context;
+			Form.Model = model;
+			
+		}
 
-        private void CloseDialogCommandBinding_Executed(object sender, ExecutedRoutedEventArgs e)
-        {
-            DialogResult = e.Parameter as bool?;
-            Close();
-        }
+		private void CloseDialogCommandBinding_Executed(object sender, ExecutedRoutedEventArgs e)
+		{
+			DialogResult = e.Parameter as bool?;
+			Close();
+		}
```

Comment:
The whole codebase uses 4 space indentation whereas your editor has inserted tabs.

Reply:
Sorry for that, seems that my Visuas studio has other settings here